todocore/src/main/java/com/knighttodo/todocore/exception/UpdateOrderNumberException.java

```diff
@@ -0,0 +1,6 @@
+package com.knighttodo.todocore.exception;
+
+public class UpdateOrderNumberException extends RuntimeException{
```

Comment:
autoformat , no space before the last symbol in the line

Reply:
Done